File: examples/id/types/go/listTheTypesOfIdsAvailable.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/id"
)

// List the types of IDs available. No query params needed.
func ListTheTypesOfIdsAvailable() {
	idService := id.NewIdService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := idService.Types(&id.TypesRequest{})
	fmt.Println(rsp, err)
}

File: examples/id/types/node/listTheTypesOfIdsAvailable.js
import * as id from "m3o/id";

// List the types of IDs available. No query params needed.
async function ListTheTypesOfIdsAvailable() {
  let idService = new id.IdService(process.env.MICRO_API_TOKEN);
  let rsp = await idService.types({});
  console.log(rsp);
}

await ListTheTypesOfIdsAvailable();
